src/V3GraphAlg.cpp

```diff
@@ -280,8 +280,9 @@ class GraphAlgRank final : GraphAlg<> {
         // If larger rank is found, assign it and loop back through
         // If we hit a back node make a list of all loops
         if (vertexp->user() == 1) {
-            m_graphp->reportLoops(m_edgeFuncp, vertexp);
-            m_graphp->loopsMessageCb(vertexp);
+            vertexp->unlinkDelete(m_graphp);
+           // m_graphp->reportLoops(m_edgeFuncp, vertexp);
```

Comment:
Should we uncomment this code at some point?

Reply:
Yes, I'm currently changing this implementation. I found that changing this so simply wouldn't work
Cause with this implementation we wouldn't be able to distinguish from real cycles.